examples/tutorials/tutor.py

```diff
@@ -187,13 +188,13 @@
 #  Returns the contents of a specified text file (or None):
 #-------------------------------------------------------------------------------
 
-def read_file ( path, mode = 'rb' ):
+def read_file ( path, mode = 'rU' ):
     """ Returns the contents of a specified text file (or None).
         """
     fh = result = None
 
     try:
-        fh     = file( path, mode )
+        fh     = open( path, mode )
```

Comment:
Can we fix this ugly set of `try..except`  blocks with a with context. :grinning: